varats-core/varats/experiment/experiment_util.py

```diff
@@ -244,6 +249,21 @@ class VersionExperiment(Experiment):  # type: ignore
     """Base class for experiments that want to analyze different project
     revisions."""
 
+    @classmethod
+    def __init_subclass__(
+        cls, shorthand: str, *args: tp.Any, **kwargs: tp.Any
+    ) -> None:
+        # mypy does not yet fully understand __init_subclass__()
+        # https://github.com/python/mypy/issues/4660
+        super().__init_subclass__(*args, **kwargs)  # type: ignore
+
+        cls.SHORTHAND = shorthand
+
+    @staticmethod
+    @abstractmethod
+    def shorthand() -> str:
+        """Experiment shorthand."""
```

Comment:
I guess this should work the same as with reports:

```suggestion
    @classmethod
    def shorthand(cls) -> str:
        """Experiment shorthand."""
        return cls.SHORTHAND
```